.gitpod.yml

```diff
@@ -0,0 +1,30 @@
+image:
+  file: .gitpod/gitpod.Dockerfile
+
+tasks:
+  - name: Bazel
+    before: "./.gitpod/gitpod-init.sh"
+    init: |
+      dotnet build ./Equinox.sln
+
+vscode:
+  extensions:
+    - Ionide.Ionide-fsharp@5.9.0
+    - muhammad-sammy.csharp@1.23.16
```

Comment:
Will this be maintained? 
I'd like to keep the dependencies lean.

Reply:
Yeah, this plugin is maintained: https://open-vsx.org/extension/muhammad-sammy/csharp
It's a fork of Omnisharp which uses a open source debugger instead of the proprietary one that comes with Omnisharp. The proprietary one can not be used by non-Microsoft products...